pkg/migration/aws_crs.go

```diff
@@ -88,9 +104,208 @@ metricsBindAddress: 0.0.0.0:10249`
 }
 
 func (m *awsMigrator) createKubeadmControlPlane(ctx context.Context) error {
-	// TODO
+	replicas := int32(1)
```

Comment:
for the migration, we need to always add only 1 master and than later scale if it was a 3 master cluster